README.md

```diff
@@ -60,15 +60,14 @@ We use http://www.apache.org/licenses/LICENSE-2.0
 ### Spring Integration
 
 As the spring projects
-[removed the embed mongo support in 2.7.0](https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-2.7-Release-Notes#springmongodbembeddedfeatures-configuration-property-removed)
```

Comment:
Yes, they updated the comment in their new release .. but they started removing the dependency with version 2.7.x. So i would not change that.

Reply:
@michaelmosmann Spring Boot still seems to manage dependency for Embedded MongoDB in 2.7.x here: https://github.com/spring-projects/spring-boot/blob/2.7.x/spring-boot-project/spring-boot-dependencies/build.gradle#L340-L346

Am I missing something?

Reply:
@izeye there are supporting an old version of this library .. and stopped to upgrading it to the new version (see https://github.com/flapdoodle-oss/de.flapdoodle.embed.mongo/issues/378 if you are interested in some 'fun')